files/ko/learn/javascript/asynchronous/introducing_workers/index.md

```diff
@@ -5,7 +5,7 @@ slug: Learn/JavaScript/Asynchronous/Introducing_workers
 
 {{LearnSidebar}}{{PreviousMenuNext("Learn/JavaScript/Asynchronous/Implementing_a_promise-based_API", "Learn/JavaScript/Asynchronous/Sequencing_animations", "Learn/JavaScript/Asynchronous")}}
 
-"비동기 JavaScript" 학습 과정의 마지막 글에서는 별도의 {{Glossary("Thread", "Thread")}} 실행 시 일부 작업을 실행할 수 있는 __workers__ 를 소개합니다.
+"비동기 JavaScript" 학습 과정의 마지막 글에서는 별도의 {{Glossary("Thread", "Thread")}} 실행 시 일부 작업을 실행할 수 있는 _workers_ 를 소개합니다.
```

Comment:
Hangul is not suitable for italics. The first reason is that there is no dedicated italic font, and the second reason is that the shape of the characters is (regular) square.

Therefore, follow the guide below and replace it with bold if necessary.

- https://github.com/mdn/translated-content/blob/main/docs/ko/translation-guide.md#%EA%B8%B0%EC%9A%B8%EC%9E%84%EA%BC%B4

Reply:
I changed it because it was emphasis used upstream https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/asynchronous/introducing_workers/index.md
If emphasis doesn't work for the local, then it should use the `**` style of strong emphasis

Reply:
```suggestion
"비동기 JavaScript" 학습 과정의 마지막 글에서는 별도의 {{Glossary("Thread", "Thread")}} 실행 시 일부 작업을 실행할 수 있는 **workers** 를 소개합니다.
```

As you said, I suggest it as `**`. thank you